src/test/java/com/hpe/adm/octane/ideplugins/integrationtests/IntegrationTestBase.java

```diff
@@ -16,33 +16,445 @@
+        fields.add(new StringFieldModel("password", "Welcome1"));
+        fields.add(new MultiReferenceFieldModel("roles", Collections.singletonList(roles.get(0))));
+        userEntityModel.setValues(fields);
+        OctaneProvider octaneProvider = serviceModule.getOctane();
+        Octane octane = octaneProvider.getOctane();
+        octane.entityList("workspace_users").create().entities(Collections.singletonList(userEntityModel)).execute();
+    }
+
+    /**
+     * This method will return the current user
+     *
+     * @return the user entityModel if found, null otterwise
+     */
+    public EntityModel getCurrentUser() {
+        OctaneProvider octaneProvider = serviceModule.getOctane();
+        Octane octane = octaneProvider.getOctane();
+        List<EntityModel> users = octane.entityList("workspace_users").get().execute().stream().collect(Collectors.toList());
+
+        for (EntityModel user : users) {
+            if (user.getValue("email").getValue().toString().equals(connectionSettingsProvider.getConnectionSettings().getUserName().toString())) {
+                return user;
+            }
+        }
+        return null;
+    }
+
+    /**
+     * This method will retrieve all the possible roles that can be assigned to a user
+     *
+     * @return - a list of enitityModels representing the possible roles
+     */
+    public List<EntityModel> getRoles() {
+        OctaneProvider octaneProvider = serviceModule.getOctane();
+        Octane octane = octaneProvider.getOctane();
+        return octane.entityList("user_roles").get().execute().stream().collect(Collectors.toList());
+    }
+
+    /**
+     * This method will return all the users
+     *
+     * @return - a list of users
+     */
+    public List<EntityModel> getUsers() {
+        EntityService entityService = serviceModule.getInstance(EntityService.class);
+        Set<String> mySet = new HashSet<>();
+        mySet.add("roles");
+
+        List<EntityModel> entities = new ArrayList<>(entityService.findEntities(
+                Entity.WORKSPACE_USER,
+                null,
+                mySet)
+        );
+        System.out.println(entities);
+        return entities;
+    }
+
+    /**
+     * This method will search for a user by its id
+     *
+     * @param id - user id
+     * @return null - if not found, userEntityModel if found
+     */
+    public EntityModel getUserById(long id) {
+        EntityService entityService = serviceModule.getInstance(EntityService.class);
+        try {
+            EntityModel entity = entityService.findEntity(Entity.WORKSPACE_USER, id);
+            System.out.println(entity);
+            return entity;
+        } catch (ServiceException e) {
+            e.printStackTrace();
+            logger.debug("There was an issue with retrieving the user with id " + id);
+        }
+        return null;
+    }
+
+
+    /**
+     * This method will return the first release in the list of releases
```

Comment:
please rephrase the Javadoc for methods and don't start with "This method ...."
Refer to: http://blog.joda.org/2012/11/javadoc-coding-standards.html, http://www.oracle.com/technetwork/articles/java/index-137868.html